### PR TITLE
feat: #1031 DB schema for web component config

### LINKED ISSUE #1031: Write DB schema for Web Components Config endpoint
**Background context or User story:**

_Building on the POC for Config Server, we need a schema that will support RESTful endpoints for the bookable widgets_

**Specification or Acceptance Criteria:**

Should support initial configurable options;

- Time between appointments
- Length of appointments
- Appointment type - see platform API `configuration/appointmentTypes`
- Negotiator ids (which negotiators are going to be available in each office)
- For these fields should be supported on GET, PUT, POST, DELETE requests

### PR DESCRIPTION
**Pull request by vuhuucuong**
fixes #1031 
### Supported methods:
- `GET` (get item by customerId)
- `PUT` (create new item)
- `PATCH` (update item by customerId)
- `DELETE` (delete item by customerId)

### Sample item in DynamoDB
Primary key: `customerId`

Screen Shot 2020-04-27 at 15 42 50

### CODE REVIEW

File: packages/web-components-config-server/src/routes/web-components-config/__mocks__/web-components-config.ts [hunk 1]
```diff
@@ -1,24 +1,12 @@
 export const webComponentConfig = {
-  configuration: {
-    timeIntervalMinutes: 10,
-    days: ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'],
```
Comment (willmcvay) [inline packages/web-components-config-server/src/routes/web-components-config/__mocks__/web-components-config.ts:4]: Sorry to be annoying but can we leave in days? Just realised we need this, sorry!

Comment (willmcvay) [reply packages/web-components-config-server/src/routes/web-components-config/__mocks__/web-components-config.ts:4]: Maybe save as numbers so we can use with dayjs easily?

Comment (vuhuucuong) [reply packages/web-components-config-server/src/routes/web-components-config/__mocks__/web-components-config.ts:4]: Sorry, do you mean to leave it in mock or what?

Comment (willmcvay) [reply packages/web-components-config-server/src/routes/web-components-config/__mocks__/web-components-config.ts:4]: Sorry not clear from the file - I meant to leave days as a schema item. I had missed it in when I wrote the ticket - as I say, my bad

**Comment by vuhuucuong** [comment]
Hey @willmcvay , I've added `daysOfWeek` as a schema field, please help me to review again

**Pull request merged**